src/sentry/tagstore/legacy/backend.py

```diff
@@ -114,12 +114,7 @@ def _get_base_qs():
                 cache.set(key, result, 60)
             return result
 
-        qs = _get_base_qs()
-
-        if status:
-            qs = qs.filter(status=status)
```

Comment:
This is done inside `_get_base_qs()` -- just minor cleanup I found while debugging.